src/components/logs-query-input.tsx

```diff
@@ -48,13 +34,13 @@ export const LogsQueryInput: React.FC<LogsQueryInputProps> = ({
       className="co-logs-expression-input"
       data-test={TestIds.LogsQueryInput}
     >
-      <SearchInput
+      <TextArea
         className="co-logs-expression-input__searchInput"
         placeholder="LogQL Query"
-        value={searchValue}
+        value={value}
         onChange={handleSearchChange}
```

Comment:
We could just pass `onChange={onChange}` here, removing the need for `handleSearchChange`